File: database/src/main/kotlin/com/tsobu/ona/database/entities/dataval/PpTzWaP6Entity.java
package com.tsobu.ona.database.entities.dataval;

import lombok.Data;
import lombok.EqualsAndHashCode;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "dataval_pp_tz_wa_p6")
@Data
@EqualsAndHashCode(callSuper = true)
public class PpTzWaP6Entity extends com.tsobu.ona.database.entities.BaseEntity implements {

    @Column(name = "index_p6")
    private Integer indexP6;

    @Column(name = "weed_pic_p6")
    private String weedPicP6;

    @Column(name = "weed_cover_p6")
    private String weedCoverP6;

    @Column(name = "cassava_height_p6")
    private String cassavaHeightP6;

    @Column(name = "weeds_height_p6")
    private String weedsHeightP6;

    @Column(name = "parent_key")
    private String parentKey;

    @Column(name = "control_key")
    private String controlKey;

    @Column(name = "set_of_weed_assessment_p6")
    private String setOfWeedAssessmentP6;

}

File: database/src/main/kotlin/com/tsobu/ona/database/entities/dataval/PpTzWaP6Entity.kt
package com.tsobu.ona.database.entities.dataval

import com.tsobu.ona.database.entities.BaseEntity
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@Table(name = "dataval_pp_tz_wa_p6")
class PpTzWaP6Entity : BaseEntity() {
    @Column(name = "index_p6")
    var indexP6: Int? = null

    @Column(name = "weed_pic_p6")
    var weedPicP6: String? = null

    @Column(name = "weed_cover_p6")
    var weedCoverP6: String? = null

    @Column(name = "cassava_height_p6")
    var cassavaHeightP6: String? = null

    @Column(name = "weeds_height_p6")
    var weedsHeightP6: String? = null

    @Column(name = "parent_key")
    var parentKey: String? = null

    @Column(name = "control_key")
    var controlKey: String? = null

    @Column(name = "set_of_weed_assessment_p6")
    var setOfWeedAssessmentP6: String? = null
}